docs/ReleaseNotes.md

```diff
@@ -18,6 +18,22 @@ The included licenses apply to the following files:
 
 ## Changelog
 
+### Version 1.8.2407
+
+This cumulative release contains numerous bug fixes and stability improvments:
```

Comment:
Since this isn't a comprehensive list, yet looks like a change list rather than a summary, should we at least say something like "here are some highlights"?

Reply:
Updated.